### ISSUE TITLE
Compatibility request for linux 4.19.20-sunxi64

### ISSUE THREAD

**Issue by hydro-b** [zfsonlinux/spl#715]
I get the following error while building spl 0.7.5 for linux 4.19.20-sunxi64 (aarch64) on Armbian (bionic):
```
/var/lib/dkms/spl/0.7.5/build/module/spl/spl-vnode.c: In function âvn_getattrâ:
/var/lib/dkms/spl/0.7.5/build/module/spl/spl-vnode.c:522:24: error: incompatible types when assigning to type âstruct timespecâ from type âstruct timespec64â
  vap->va_atime         = stat.atime;
                        ^
/var/lib/dkms/spl/0.7.5/build/module/spl/spl-vnode.c:523:24: error: incompatible types when assigning to type âstruct timespecâ from type âstruct timespec64â
  vap->va_mtime         = stat.mtime;
                        ^
/var/lib/dkms/spl/0.7.5/build/module/spl/spl-vnode.c:524:24: error: incompatible types when assigning to type âstruct timespecâ from type âstruct timespec64â
  vap->va_ctime         = stat.ctime;
                        ^
scripts/Makefile.build:303: recipe for target '/var/lib/dkms/spl/0.7.5/build/module/spl/spl-vnode.o' failed
make[5]: *** [/var/lib/dkms/spl/0.7.5/build/module/spl/spl-vnode.o] Error 1
scripts/Makefile.build:544: recipe for target '/var/lib/dkms/spl/0.7.5/build/module/spl' failed
make[4]: *** [/var/lib/dkms/spl/0.7.5/build/module/spl] Error 2
make[4]: *** Waiting for unfinished jobs....
  CC [M]  /var/lib/dkms/spl/0.7.5/build/module/splat/splat-list.o
In file included from /var/lib/dkms/spl/0.7.5/build/include/sys/vnode.h:44:0,
                 from /var/lib/dkms/spl/0.7.5/build/include/sys/kobj.h:28,
                 from /var/lib/dkms/spl/0.7.5/build/module/splat/splat-kobj.c:27:
/var/lib/dkms/spl/0.7.5/build/include/sys/sunldi.h:35:0: warning: "SECTOR_SIZE" redefined
 #define SECTOR_SIZE 512
 
In file included from /var/lib/dkms/spl/0.7.5/build/include/sys/uio.h:30:0,
                 from /var/lib/dkms/spl/0.7.5/build/include/sys/vnode.h:42,
                 from /var/lib/dkms/spl/0.7.5/build/include/sys/kobj.h:28,
                 from /var/lib/dkms/spl/0.7.5/build/module/splat/splat-kobj.c:27:
./include/linux/blkdev.h:1030:0: note: this is the location of the previous definition
 #define SECTOR_SIZE (1 << SECTOR_SHIFT)
 
  CC [M]  /var/lib/dkms/spl/0.7.5/build/module/splat/splat-generic.o
  CC [M]  /var/lib/dkms/spl/0.7.5/build/module/splat/splat-cred.o
  CC [M]  /var/lib/dkms/spl/0.7.5/build/module/splat/splat-zlib.o
In file included from /var/lib/dkms/spl/0.7.5/build/include/sys/sunddi.h:30:0,
                 from /var/lib/dkms/spl/0.7.5/build/module/splat/splat-generic.c:27:
/var/lib/dkms/spl/0.7.5/build/include/sys/sunldi.h:35:0: warning: "SECTOR_SIZE" redefined
 #define SECTOR_SIZE 512
 
In file included from /var/lib/dkms/spl/0.7.5/build/include/sys/uio.h:30:0,
                 from /var/lib/dkms/spl/0.7.5/build/include/sys/sunddi.h:29,
                 from /var/lib/dkms/spl/0.7.5/build/module/splat/splat-generic.c:27:
./include/linux/blkdev.h:1030:0: note: this is the location of the previous definition
 #define SECTOR_SIZE (1 << SECTOR_SHIFT)
 
  CC [M]  /var/lib/dkms/spl/0.7.5/build/module/splat/splat-linux.o
  LD [M]  /var/lib/dkms/spl/0.7.5/build/module/splat/splat.o
Makefile:1520: recipe for target '_module_/var/lib/dkms/spl/0.7.5/build/module' failed
make[3]: *** [_module_/var/lib/dkms/spl/0.7.5/build/module] Error 2
make[3]: Leaving directory '/usr/src/linux-headers-4.19.20-sunxi64'
Makefile:11: recipe for target 'modules' failed
make[2]: *** [modules] Error 2
make[2]: Leaving directory '/var/lib/dkms/spl/0.7.5/build/module'
Makefile:589: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/var/lib/dkms/spl/0.7.5/build'
Makefile:470: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by behlendorf** [comment]
@hydro-b these build errors should be resolved in the latest 0.7.13 tag.

**Comment by hydro-b** [comment]
@behlendorf Yup, spl compiled fine now \o/ (... and zfs as well for the record). Happily running zol on pine64 now.
